src/asmcnc/production/z_head_qc_jig/z_head_qc_b4_apr_21.py

```diff
@@ -585,8 +586,7 @@ def update_status_text(self, dt):
         self.consoleStatusText.text = self.sm.get_screen('home').gcode_monitor_widget.consoleStatusText.text
 
     def do_reboot(self):
```

Comment:
Why have this function in the first place? just call console_utils.reboot() directly

Reply:
please call console_utils directly